src/ts/components/Container.ts

```diff
@@ -113,7 +113,7 @@ export class Container<Config extends ContainerConfig> extends Component<Config>
   /**
    * Updates the DOM of the container with the current components.
    */
-  protected updateComponents(): void {
+  updateComponents(): void {
```

Comment:
How about adding the intended use-case for manual calls? Adding docs here would help distinguish between automatic and manual calls


```suggestion
/**
  * Updates the DOM of the container with the current components.
  * 
  * This is called automatically after construction. However, when you dynamically
  * add or remove components at runtime you must call `updateComponents()` to
  * re-render the container’s children.
  * Use only in those cases, it is not needed during normal UI lifecycle.
  */
  updateComponents(): void {
```

Reply:
[cb31f17](https://github.com/bitmovin/bitmovin-player-ui/pull/746/commits/cb31f176e40d451717e646f1c8101b1daf8131ee)